unistore.js

```diff
@@ -30,8 +30,7 @@ export function createStore(state) {
 
 	function setState(update, overwrite) {
 		state = overwrite ? update : assign(assign({}, state), update);
-		let currentListeners = listeners;
```

Comment:
This line is important, it retains a variable reference locally that is replaced in outer scope.

Reply:
That's true!